website/pages/docs/builders/vmware/vsphere-clone.mdx

```diff
@@ -45,6 +46,25 @@ necessary for this build to succeed and can be found further down the page.
 
 @include 'builder/vsphere/common/ConfigParamsConfig-not-required.mdx'
 
+### Boot configuration
+
+@include 'common/bootcommand/BootConfig.mdx'
```

Comment:
This is not the compatible documentation but soon it will be after https://github.com/hashicorp/packer/pull/9406 is merged.

Reply:
Then this is fixed now